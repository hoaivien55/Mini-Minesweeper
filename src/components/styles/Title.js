import styled from "styled-components";

const H4 = styled.h4`
    font-size: 50px;

`;

export { H4 };
