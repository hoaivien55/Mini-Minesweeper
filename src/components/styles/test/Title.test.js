import renderer from "react-test-renderer";
import { H4 } from "../Title";

it("H4: Virtual DOM snapshot", () => {
	const tree = renderer.create(<H4 />).toJSON();
	expect(tree).toMatchSnapshot();
});
