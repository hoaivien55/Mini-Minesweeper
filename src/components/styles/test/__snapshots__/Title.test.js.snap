// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`H4: Virtual DOM snapshot 1`] = `
.c0 {
  font-size: 50px;
}

<h4
  className="c0"
/>
`;
